File: C/Iniciante/1101sequenciadenumerosesoma.c
#include <stdio.h>
int main() {
    int m,n,soma,mm,mn,i;
    scanf("%d%d",&m,&n);
    while(1){
        if(m<=0 || n<=0)break;
        if(m<n){
            mm=m;
            mn=n;
        }else{
            mm=n;
            mn=m;
        }
        soma=0;
        for(i=mm;i<=mn;i++){
            printf("%d ",i);
            soma+=i;
        }
        printf("Sum=%d\n",soma);
        scanf("%d%d",&m,&n);
    }
    return 0;
}

File: Python/iniciante/1101sequenciadenumerosesoma.py
x, y = input().split(" ")
x = int(x)
y = int(y)
while 1:
    if x <= 0 or y <= 0:
        break
    if x < y:
        mx = x
        my = y
    else:
        mx = y
        my = x
    soma = 0
    for i in range(mx, my + 1):
        print("%d " % i, end="")
        soma += i
    print("Sum=%d\n" % soma)
    x, y = input().split(" ")
    x = int(x)
    y = int(y)
